# Combined system info message

Header          header

string          hostname

BatteryStatus   battery_base
BatteryStatus   battery_pc
NetworkStatus   network
SystemStatus    system
